freiburg/services.md

```diff
@@ -17,47 +17,75 @@ The Freiburg Galaxy Team is offering several services to enable **reproducible**
 
 ---
 <a name="training"></a>
 # <i class="fa fa-graduation-cap"></i> Training
 
-Supporting researchers by educating them in *e.g.* big data analysis, programming, data management, and Galaxy server administration, is one of our goals as we believe that **sharing of knowledge** and the **open science** movement are the key points in future.
+We are passionate about training! Our team wants to support researchers to take part in their own data analyses by educating them in *e.g.* big data analysis, programming, data management, and Galaxy server administration. We believe that **sharing of knowledge** and the **open science** movement are the key points in future.
 
 <div class="multiple-img" style="text-align: right;">
     <img src="/assets/media/training_1.jpg" width="200px" alt="Workshop picture" />
     <img src="/assets/media/training_2.jpg" width="200px" alt="Workshop picture" />
     <img src="/assets/media/training_3.jpg" width="200px" alt="Workshop picture" />
 </div>
 
-### Workshops
+### Galaxy Training Workshops
+
+Locally in Freiburg, we offer twice per year a free full-week hands-on high-throughput sequencing data analysis workshop. A typical workshop schedule looks as follows: 
+
+- Monday:     Introduction to Galaxy and HTS
+- Tuesday:    Quality control, IGV, ChIP-Seq, 
+- Wednesday:  RNA-Seq 
+- Thursday:   HiC data analysis, Methyl-C data analysis
+- Friday:     Bring your own data, exercises
+
+Topice usually are selected from desired topics of the applicants. 
+
+All workshops are announced on our [events page](https://galaxyproject.eu/freiburg/events). Registrations are possible through our website.
+
+We and other members of the Galaxy training network (GTN) are giving training courses around the world for data analysis and for developers and administrators. Check the website of the main [Galaxy project](https://galaxyproject.org/events) as well as the [de.NBI](https://www.denbi.de) website for [de.NBI courses](https://www.denbi.de/training)
 
-In Freiburg, we already offer twice per year a full-week hands-on high-throughput sequencing data analysis workshop: Introduction to Galaxy and HTS, RNA-Seq, ChIP-Seq, Exome-Seq, MethylC-Seq, etc. Registrations are possible through our website.
+### Galaxy Training Material
 
-We are also giving training around the world on demand: for data analysis but also for developers and administrators.
+The Freiburg Galaxy Team is the main contributor to the community-driven development of GTN Galaxy training material. All of this material is **online, and freely accessible for everyone** at [https://training.galaxyproject.org](https://training.galaxyproject.org). 
```

Comment:
""The Freiburg Galaxy Team is the main contributor" --> I think we should not mention ourself here that strong, maybe that pushes others back in contributing?

Reply:
is one of the main contributors.

Sorry, of course. This before was the old text